docs/content/install.sh

```diff
@@ -0,0 +1,104 @@
+
```

Comment:
Yes, sorry - I thought that since we provide the script as an input to bash directly this line will be redundant, but was probably mistaken.
I added `set -e` as well.